formatting.md

```diff
@@ -20,15 +20,24 @@ developers may find it more productive to:
 The alternative is often tiring formatting debates during code reviews, where
 time may be better spent on code behavior rather than code style.
 
-### Automatically formatting code in IntelliJ
+### Automatically formatting code in Android Studio, and IntelliJ
 
-Automatic formatting of code is supported in IntelliJ if you have the
-`Dart` plugin (see [IntelliJ setup](/ide-setup/)).
+Install the the `Dart` plugin (see [Editor setup](/get-started/editor/)
+to enable automatic formatting of code in Android Studio and IntelliJ.
 
 To automatically format the code in the current source code window, right-click
 in the code window and select `Reformat with Dart style`. You can add a keyboard
 shortcut to this in Keymap section of IntelliJ Preferences.
 
+### Automatically formatting code in VS Code
+
+Install the `Dart-Code` plugin (see [Editor setup](/get-started/editor/)
```

Comment:
This one has unclosed paren too (also, it suggests editor page has a section in enabling automatic formatting, but I can't see it; though the next paragraph explains how to sort-of-automatically-but-not-on-type format).

Reply:
Fidex